src/topology/continuous_on.lean

```diff
@@ -337,6 +337,35 @@ begin
   exact (hf x hx).mem_closure_image hx
 end
 
+theorem is_open_map.continuous_on_range_of_left_inverse {f : α → β} (hf : is_open_map f)
```

Comment:
Can't you deduce this one from the next one, by taking `s = univ`? There will be some fiddling around since `f` and `(λ x : s, f x)` are not the same thing, but hopefully it shouldn't be too bad.